lib/Runtime/DetachedStateBase.h

```diff
@@ -20,6 +20,10 @@ namespace Js
         {
         }
 
+        virtual ~DetachedStateBase()
+        {
+        }
+
```

Comment:
Since this whole method could be a single line, I think it would be better to have the braces on the same line, like you did with `~InterruptPoller`